src/commands/veracode_prepare_artifacts.yml

```diff
@@ -19,6 +19,7 @@ steps:
       name: Build temp docker image and copy app files
       no_output_timeout: 20m
       command: |
+        export DOCKER_BUILDKIT=1
```

Comment:
Some projects are now using `buildx` which uses the buildkit docker builder instead.  The docker files in these projects then aren't compatible with the legacy builder (`BUILDPLATFORM` not defined) so easiest thing to do is to switch all projects to using buildkit.  It is backwards compatible with the legacy builder and is apparently a lot faster too...